src/state-summary/state-card-display.html

```diff
@@ -40,7 +40,15 @@
   }
 
   computeStateDisplay(stateObj) {
-    return window.hassUtil.computeStateState(stateObj);
+    switch (window.hassUtil.computeDomain(stateObj)) {
+      case 'zwave':
```

Comment:
Move this to window.hassUtil.computeStateState

Reply:
Also include stage for "Sleeping", so everything except "ready" will have stage

Reply:
The way we have it configured in the backend right now is that Sleeping will only be sent after the query stage is Complete.